src/main/java/io/percy/appium/metadata/AndroidMetadata.java

```diff
@@ -96,4 +96,28 @@ public String getDisplaySysDump() {
         return (String) Cache.CACHE_MAP.get("getDisplaySysDump_" + sessionId);
     }
 
+
+    public String orientation() {
+      if (orientation != null) {
+          if (orientation.toLowerCase().equals("portrait") || orientation.toLowerCase().equals("landscape")) {
+              return orientation.toLowerCase();
+          } else if (orientation.toLowerCase().equals("auto")) {
+              try {
+                  return driver.getOrientation().toString().toLowerCase();
+              } catch (java.lang.NoSuchMethodError e) {
+                  return "portrait";
+              }
+          } else {
+              return "portrait";
+          }
+      } else {
+          Object orientationCapability = driver.getCapabilities().getCapability("orientation");
+          if (orientationCapability != null) {
```

Comment:
Why have we moved this function from metadata to here? It seems repetative.

Reply:
also take master merge

Reply:
in v8 getOrientation is not on parent lvl. its moved to respective classes.

Reply:
Ack, Can you add a comment. above